build-output-api/README.md

```diff
@@ -16,6 +16,7 @@ that are listed below.
 - [Edge Functions](./edge-functions)
 - [Edge Middleware](./edge-middleware)
 - [Image Optimization](./image-optimization)
+- [On-demand Incremental Static Regeneration (ISR) Revalidation](./on-demand-isr-revalidation)
```

Comment:
```suggestion
- [On-demand Incremental Static Regeneration (ISR)](./on-demand-isr-revalidation)
```
The revalidation here seems redundant, I don't think we describe it this way anywhere in the docs.

Reply:
I was confused by this reading the docs here: https://vercel.com/docs/concepts/next.js/incremental-static-regeneration

The phrase "on demand" is used throughout, regardless of the revalidation case. This situation is called "on-demand Incremental Static Regeneration" once on that page, but it's not completely titlecased or bolded. The heading for that section is called "On-Demand Revalidation".

Should this docs page be updated to make this clearer? I'm happy to rename this to whatever makes sense.

Reply:
Might be good to get thoughts in `#devex` or `#marketing` but `regeneration` and `revalidation` are pretty much synonyms in this case the difference is between on-demand and default ISR would be time-based vs on-demand (manually triggered).

We should probably also rename the example folder to drop the extra `-revalidation` at the end. For prior art our current on-demand ISR example can be seen here https://on-demand-isr.vercel.app/ (https://github.com/leerob/on-demand-isr)

Reply:
Agree with @ijjk's points there. There are two modes:

- Normal - time based revalidation/regeneration
- On-demand - push based revalidation/regeneration

Reply:
@leerob @ijjk That sounds very reasonable to me.

I still think the docs are misleading about this. Should those be revised? https://vercel.com/docs/concepts/next.js/incremental-static-regeneration